examples/async-examples.js

```diff
@@ -0,0 +1,59 @@
+(async () => {	
+	const revai = require('../../revai-node-sdk');
+
+	/* Initialize your client with your revai access token */
+	const token = '<YOUR-ACCESS-TOKEN>';
```

Comment:
+1 for using a config file both for development and for publishing to prevent us from accidentally committing access_tokens to feature branches while developing

Reply:
One idea is to have a public sample access token that actually works, but is heavily throttled. I don't support the idea.

Reply:
Using a config file is a very good idea. I'm not sure where we would expect them to put the file though. I'm sure other node packages do this so I'll look into what they do.

I definitely don't support the throttled public token either